tests/functional/Agent.php

```diff
@@ -500,5 +500,75 @@ public function testStaleActions()
         $this->boolean($item->getFromDB($item->fields['id']))->isTrue();
         $this->integer($item->fields['is_deleted'])->isIdenticalTo(1);
         $this->integer($item->fields['states_id'])->isIdenticalTo($states_id);
+
+        //create another new status
```

Comment:
As far as I understand the original need; we should ensure state is not changed on agent that have a state not configured to be changed